babel.config.js

```diff
@@ -25,18 +9,12 @@ const environments = {
   browsers,
 };
 
-module.exports = function (api /* : ApiType */) {
-  api.assertVersion(7);
-  api.cache.forever();
-
-  return {
-    presets:
-      target === "storybook" ? [] : [["@adeira/babel-preset-adeira", { target, environments }]],
-    plugins: ["babel-plugin-styled-components"],
-    env: {
-      test: {
-        plugins: ["require-context-hook"],
-      },
+module.exports = {
+  presets: [["@adeira/babel-preset-adeira", { target, environments }]],
+  plugins: ["babel-plugin-styled-components"],
+  env: {
+    test: {
+      plugins: ["require-context-hook"],
     },
-  };
+  },
 };
```

Comment:
I simplified this, now this serves as the base preset which packages extend as needed, e.g. `orbit-components` modifies it if `BABEL_TARGET` is `storybook`.